packages/@sanity/migrate/src/fs-webstream/__test__/partial.ndjson

```diff
@@ -0,0 +1,11 @@
+{"foo": 0,"bar": 0, "baz": 0}
+{"foo": 1,"bar": 1, "baz": 1}
+{"foo": 2,"bar": 2, "baz": 2}
+{"foo": 3,"bar": 3, "baz": 3}
+{"foo": 4,"bar": 4, "baz": 4}
+{"foo": 5,"bar": 5, "baz": 5}
+{"foo": 6,"bar": 6, "baz": 6}
+{"foo": 7,"bar": 7, "baz": 7}
+{"foo": 8,"bar": 8, "baz": 8}
+{"foo": 9,"bar": 9, "baz": 9}
+{"foo": 10,
```

Comment:
this may look like a bug, but is working as intended. When the returned stream is cancelled, we also want to stop the buffering.